src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/AdvSimd.PlatformNotSupported.cs

```diff
@@ -106,6 +106,224 @@ public new abstract class Arm64 : ArmBase.Arm64
+            // /// </summary>
+            // public static Vector64<double> CompareLessThanOrEqual(Vector64<double> left, Vector64<double> right) { throw new PlatformNotSupportedException(); }
+
+            // /// <summary>
+            // /// int64x1_t vcle_s64 (int64x1_t a, int64x1_t b)
+            // ///   A64: CMHS Vd, Vn, Vm
+            // /// </summary>
+            // public static Vector64<long> CompareLessThanOrEqual(Vector64<long> left, Vector64<long> right) { throw new PlatformNotSupportedException(); }
+
+            // /// <summary>
+            // /// uint64x1_t vcle_u64 (uint64x1_t a, uint64x1_t b)
+            // ///   A64: CMHS Vd, Vn, Vm
+            // /// </summary>
+            // public static Vector64<ulong> CompareLessThanOrEqual(Vector64<ulong> left, Vector64<ulong> right) { throw new PlatformNotSupportedException(); }
+
+            /// <summary>
+            /// float64x2_t vcleq_f64 (float64x2_t a, float64x2_t b)
+            ///   A64: FCMGT Vd, Vn, Vm
+            /// </summary>
+            public static Vector128<double> CompareLessThanOrEqual(Vector128<double> left, Vector128<double> right) { throw new PlatformNotSupportedException(); }
+
+            /// <summary>
+            /// int64x2_t vcleq_s64 (int64x2_t a, int64x2_t b)
+            ///   A64: CMHS Vd, Vn, Vm
+            /// </summary>
+            public static Vector128<long> CompareLessThanOrEqual(Vector128<long> left, Vector128<long> right) { throw new PlatformNotSupportedException(); }
+
+            /// <summary>
+            /// uint64x2_t vcleq_u64 (uint64x2_t a, uint64x2_t b)
+            ///   A64: CMHS Vd, Vn, Vm
+            /// </summary>
+            public static Vector128<ulong> CompareLessThanOrEqual(Vector128<ulong> left, Vector128<ulong> right) { throw new PlatformNotSupportedException(); }
+
+            // /// <summary>
+            // /// float64x1_t vtst_f64 (float64x1_t a, float64x1_t b)
+            // ///   A64: CMTST Dd, Dn, Dm
+            // /// The above native signature does not exist. We provide this additional overload for consistency with the other scalar APIs.
+            // /// </summary>
+            // public static Vector64<double> CompareTest(Vector64<double> left, Vector64<double> right) { throw new PlatformNotSupportedException(); }
+
+            // /// <summary>
+            // /// int64x1_t vtst_s64 (int64x1_t a, int64x1_t b)
+            // ///   A64: CMTST Vd, Vn, Vm
+            // /// </summary>
+            // public static Vector64<long> CompareTest(Vector64<long> left, Vector64<long> right) { throw new PlatformNotSupportedException(); }
+
+            // /// <summary>
+            // /// uint64x1_t vtst_u64 (uint64x1_t a, uint64x1_t b)
+            // ///   A64: CMTST Vd, Vn, Vm
+            // /// </summary>
+            // public static Vector64<ulong> CompareTest(Vector64<ulong> left, Vector64<ulong> right) { throw new PlatformNotSupportedException(); }
```

Comment:
Curious about the commented out code, is this because there's no support for `x1_t` vectors?

Reply:
Yes, that's right